src/nuclio/projects/project/functions/function-collapsing-row/function-collapsing-row.component.js

```diff
@@ -190,5 +202,69 @@
 
             NuclioHeaderService.updateMainHeader('Projects', ctrl.title, $state.current.name);
         }
+
+        /**
+         * Show dialog with YAML function config
+         */
+        function viewConfig() {
+            var sourceCode = ExportService.getFunctionConfig(ctrl.function);
+            var label = 'Config ' + ctrl.title.function;
+
+            ngDialog.open({
+                template:
+                    '<div class="view-yaml-dialog-container">' +
+                        '<div class="view-yaml-dialog-header">' +
+                            '<div class="title">' + label + '</div>' +
+                            '<div class="copy-to-clipboard" ' +
+                                'data-ng-click="ngDialogData.copyToClipboard(ngDialogData.sourceCode)" data-uib-tooltip="Copy to clipboard" ' +
+                                'data-tooltip-placement="left" data-tooltip-popup-delay="300" data-tooltip-append-to-body="true">' +
+                                '<div class="ncl-icon-copy"></div>' +
+                            '</div>' +
+                            '<div class="close-button igz-icon-close" data-ng-click="closeThisDialog(0)"></div>' +
+                        '</div>' +
+                        '<div class="main-content">' +
+                            '<ncl-monaco class="monaco-code-editor" ' +
+                                'data-function-source-code="ngDialogData.sourceCode" ' +
+                                'data-mini-monaco="false" ' +
+                                'data-selected-theme="ngDialogData.editorTheme" ' +
+                                'data-language="yaml" ' +
+                                'data-read-only="true">' +
+                            '</ncl-monaco>' +
+                        '</div>' +
+                        '<div class="buttons">' +
+                            '<button class="igz-button-primary" tabindex="0" data-ng-click="closeThisDialog(0)">Close</button>' +
+                        '</div>' +
+                    '</div>',
```

Comment:
Please move this HTML template to an .tpl.html file and here put its path, for example:
```js
template: 'path/to/file.tpl.html'
```